src/Microsoft.Dnx.Tooling/List/LibraryDependencyFlatRenderer.cs

```diff
@@ -50,28 +50,28 @@ public IEnumerable<string> GetRenderContent(IGraphNode<LibraryDescription> root)
             return results;
         }
 
-        private IDictionary<LibraryDescription, ISet<LibraryDescription>> FindImmediateDependent(IGraphNode<LibraryDescription> root)
+        private IDictionary<LibraryResolution, ISet<LibraryResolution>> FindImmediateDependent(IGraphNode<LibraryResolution> root)
         {
-            var result = new Dictionary<LibraryDescription, ISet<LibraryDescription>>();
+            var result = new Dictionary<LibraryResolution, ISet<LibraryResolution>>();
 
-            root.DepthFirstPreOrderWalk(
-                visitNode: (node, ancestors) =>
+            IGraphNodeExtensions.DepthFirstPreOrderWalk<Runtime.LibraryResolution>(
+root,                visitNode: (Func<IGraphNode<Runtime.LibraryResolution>, IEnumerable<IGraphNode<Runtime.LibraryResolution>>, bool>)((IGraphNode<Runtime.LibraryResolution> node, IEnumerable<IGraphNode<Runtime.LibraryResolution>> ancestors) =>
```

Comment:
I think this was changed by a refactoring. I've seen it before. Renaming a type somehow triggered VS to convert the extension method into static method calls